mcp_servers/app/main.py

```
@@ -1314,3 +1314,249 @@ async def text_to_graphql_query(question: str, network: str = "mainnet") -> Dict
+        5. Return: "GIB (token 0.0.XXXXX) is trading at $X.XX USD"
+    """
+    try:
+        logger.info(f"🔍 FIND_TOKEN_BY_NAME TOOL: Searching for token '{token_name}' on {network}")
+
+        # Use GraphQL to search for token by name/symbol
+        gql_service = get_graphql_service()
+
+        # Craft a question to search for the token
+        question = f"Find token with name or symbol '{token_name}'"
+
+        logger.info(f"🚀 FIND_TOKEN_BY_NAME TOOL: Querying GraphQL for token search")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)